cdap-watchdog/src/main/java/co/cask/cdap/logging/remote/DistributedLogProcessor.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.logging.remote;
+
+import java.util.Iterator;
+
+/**
+ * The {@link RemoteExecutionLogProcessor} used in Distributed mode.
+ */
+public class DistributedLogProcessor implements RemoteExecutionLogProcessor {
+
+  @Override
+  public void process(Iterator<byte[]> loggingEventBytes) {
+    // TODO (CDAP-13380): implement distributed log processor
```

Comment:
Still a TODO??

Reply:
Ah, the implementation doesn't use this class. I will remove this class, as it is unused.